models/case.go

```diff
@@ -39,7 +46,7 @@ type CreateCaseAttributes struct {
 
 type UpdateCaseAttributes struct {
 	Id          string
-	DecisionIds []string
+	DecisionIds []string // Decision Ids to add to the case - not list of decision ids to replace on the case
```

Comment:
If this is confusing, you may pick an explcit prop name like `DecisionToAddIds`

Reply:
true 👍  good point

Reply:
ok turned out it's even better, it's not even the struct I was really using for the update: this field was never actually used anywhere (I did the rename trick on another variable though, cf last commit)